frontend/src/environments/environment.prod.ts

```diff
@@ -1,12 +1,12 @@
 export const environment = {
   production: true,
   firebaseConfig: {
-    apiKey: "AIzaSyCmsP1M4ie5rx1BYPwpPWhsalw6LpHcrMw",
-    authDomain: "ck-board.firebaseapp.com",
-    projectId: "ck-board",
-    storageBucket: "ck-board.appspot.com",
-    messagingSenderId: "473282317820",
-    appId: "1:473282317820:web:0bc303a476e0f514153679",
-    measurementId: "G-00CHHWR3SW"
-  }
+    apiKey: 'AIzaSyCmsP1M4ie5rx1BYPwpPWhsalw6LpHcrMw',
+    authDomain: 'ck-board.firebaseapp.com',
+    projectId: 'ck-board',
+    storageBucket: 'ck-board.appspot.com',
+    messagingSenderId: '473282317820',
+    appId: '1:473282317820:web:0bc303a476e0f514153679',
+    measurementId: 'G-00CHHWR3SW',
```

Comment:
I noticed on the backend, its formatting to double quotes, but on the frontend its using single quotes. Not sure if its intended but would be better to keep it consistent.